sopel/config/core_section.py

```diff
@@ -712,24 +991,51 @@ def homedir(self):
     """The number of seconds acceptable between pings before timing out.
 
     :default: ``120``
+
+    You can increase the timout (for example up to 200s) like this:
+
+    .. code-block:: ini
+
+        timeout = 200
+
     """
 
     use_ssl = ValidatedAttribute('use_ssl', bool, default=False)
     """Whether to use a SSL/TLS encrypted connection.
 
     :default: ``False``
+
+    Example with SSL on:
+
+    .. code-block:: ini
+
+        use_ssl = yes
```

Comment:
```suggestion
        use_ssl = True
```

Same editorial comment as `log_raw`.